src/data_management/rasters/RasterSourceTable.tsx

```diff
@@ -2,7 +2,6 @@ import React, { useState, useEffect } from 'react';
 import { useSelector} from 'react-redux';
```

Comment:
I refactor this RasterSourceTable component quite a bit since I want to reuse the DeleteModal and refactor the deleteAction (this work has already been done for all other tables but not this one since this case is a bit more complicated due to its related raster layers). Also, many of commented out code is no longer relevant so I removed them too.

In the old code, there is still a case of a failed deletion of raster source with a **412** status response which is not handled by the DeleteMoal. I could not reproduce this specific case so do you know how important it is and if we should still handle this case? If yes, I can continue improve this part.